ruby/vampires.rb

```diff
@@ -1,4 +1,4 @@
-puts "How many employees will be processed (please enter a number)?"
+puts "How many employees will be processed? (please enter a number)"
 employees = gets.chomp.to_i
 	number = 0
 	until number == employees
```

Comment:
For line 23

Consider checking  the birth year agains the Ruby time class `Time.now.year` this makes the code more dynamic and allows you the freedom of not having to rewrite the file with hardcoded values as years change.